src/app/reskin/components/stepper/deposit/amount.tsx

```diff
@@ -0,0 +1,22 @@
+import getEmilyLimits from "@/actions/get-emily-limits";
+import { useQuery } from "@tanstack/react-query";
+
+export function AmountDescription({}) {
+  const { data: minimumDeposit } = useQuery({
+    queryKey: ["emilyLimits"],
+    queryFn: async () => {
+      const response = await getEmilyLimits();
+      const isUnlimited = response.perDepositMinimum === 0;
+      // to handle devenv situations
+      return isUnlimited ? 0.0001 : response.perDepositMinimum / 1e8;
```

Comment:
Why the `0.0001` instead of `0` if we don't have a limit?

Reply:
arbitrary value can change it to anything assuming that we use that for testing in devnet and devenv

Reply:
Why not just zero if it's zero?

Reply:
because 0 btc is not valid amount to send 1 sat is

Reply:
I set it to 1 sat

Reply:
Not a blocker, but here we talk about the min wrt emily, so zero it's a valid number. If we are talking about valid bitcoin amount, then we should use a dust value.